lib/src/renderers/web_gl_renderer.dart

```diff
@@ -6145,7 +6147,7 @@ class WebGLRenderer implements Renderer {
         spotLength += 1;
 
       } else if (light is HemisphereLight) {
-
+        
```

Comment:
This should be removed. In fact, could you run `dartfmt --max_line_length 120` on the files you changed? I see a couple other lines that should probably be formatted.